sanic/cookies.py

```diff
@@ -104,13 +103,19 @@ def __setitem__(self, key, value):
     def encode(self, encoding):
         output = ['%s=%s' % (self.key, _quote(self.value))]
         for key, value in self.items():
-            if key == 'max-age' and isinstance(value, int):
-                output.append('%s=%d' % (self._keys[key], value))
-            elif key == 'expires' and isinstance(value, datetime):
-                output.append('%s=%s' % (
-                    self._keys[key],
-                    value.strftime("%a, %d-%b-%Y %T GMT")
-                ))
+            if key == 'max-age':
+                try:
```

Comment:
Why do you want to format max-age as an int and fallback to a string ? You should just do: `output.append('%s=%s' % (self._keys[key], value))` without any try/except.

Also the previous code and this one is not semantically the same. In previous case, if key=='max-age' and value in a str, the `if` block will not be executed.